views/celebrities/new-movies.hbs

```diff
@@ -0,0 +1,19 @@
+<h2>Create a new Movie</h2>
+
+<form action="/movies/create" method="post">
+
+  <label for="title">Title</label>
+  <input type="text" name="title" id="title" />
+
+  <label for="genre">Genre</label>
+  <input type="text" name="genre" id="genre" />
+
+  <label for="plot">Plot</label>
+  <input type="text" name="plot" id="plot" />
+
+  <label for="cast">Cast</label>
+  <input type="text" name="cast" id="cast" />
```

Comment:
Remember that this should be a select element with the options for the user to select the celebrity!